lib/models/action.rb

```diff
@@ -0,0 +1,21 @@
+require 'json'
+
+module G5HubApi
+  class Action
+    attr_accessor :label,
+                  :url
+
+    def initialize(hash = {})
+      @label  = hash['label'] || hash[:label]
+      @url    = hash['url']   || hash[:url]
+    end
+
+    def as_json(_ = {})
```

Comment:
was this unused parameter added due to backward compatibility? if not I would remove it.